inspirehep/dojson/hepnames/fields/bd1xx.py

```diff
@@ -31,7 +31,6 @@
 
 @hepnames.over('acquisition_source', '^541[10_].')
 @utils.for_each_value
-@utils.filter_values
```

Comment:
Ok. Keep in mind that commits should be **atomic**: they should deal with exactly one thing at a time, so this change should have been in a different commit.

Because of several other issues like this I will have to squash this PR to a single commit : (